product-service/model/Product.ts

```diff
@@ -0,0 +1,9 @@
+export class Product {
```

Comment:
Why would not you declare a interface instead of class?

Reply:
Ok. I agree that interface is enough for a model. Fixed.